.github/workflows/pr.yml

```diff
@@ -65,13 +65,13 @@ jobs:
         env:
           cache-name: node-modules
         with:
-          path: ~/.npm
+          path: ~/.yarn
           key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Same. This should look at `yarn.lock`

Reply:
Essentially anything that looks at `yarn.lock` might not work correctly

Reply:
Also that `path` should be [`**/node_modules`](https://github.com/newrelic/developer-website/blob/develop/.github/workflows/validate-pr.yml#L26) so that it caches the installed node modules and not the yarn cache (which doesn't seem to do much)

Reply:
@jerelmiller I can fix that in a new PR.

Reply:
@jerelmiller can you clarify what line 68 needs to be, is it `yarn.lock` or `**/node_modules`

Reply:
updated it in another PR: https://github.com/newrelic/opensource-website/pull/733